doc/man7/provider-base.pod

```diff
@@ -131,20 +142,20 @@ F<libcrypto>):
 core_gettable_params() returns a constant array of descriptor
 B<OSSL_PARAM>, for parameters that core_get_params() can handle.
 
-core_get_params() retrieves I<prov> parameters from the core.
+core_get_params() retrieves parameters from the core for the given I<handle>.
 See L</Core parameters> below for a description of currently known
 parameters.
 
 =for comment core_thread_start() TBA
 
 core_get_library_context() retrieves the library context in which the
-B<OSSL_PROVIDER> object I<prov> is stored.
+B<OSSL_CORE_HANDLE> object I<handle> is stored.
```

Comment:
`... retrieves the library context in which the library object for the current provider is stored, accessible through the I<handle>.`

Reply:
Done